supabase-case-study.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Supabase Case Study
+author: Marte Moslet <martemoslet>
+tags: supabase, case study, platform, BaaS
+---
+
+ ## Introduction
+
+ Supabase is a backend-as-a-Service (BaaS) platform that offers PostgreSQL database, authentication, storage and edge functions. It was created in 2019 as an open-source alternative to Firebase. 
+
+ ## Brief history
+
+- 2019: Supabase is created
+- July 2020: Announced Auth
+- November 2020: Supabase Beta
```

Comment:
In December 2020 Supabase received $6m in funding.

There is a great article on the history here: https://www.topofthelyne.co/p/how-supabase-is-taking-on-the-goliath

Reply:
Thanks, I have now added that fact